### ISSUE TITLE
"See more" links on the homepage are not accessible

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons#11020]
If a user of assistive tech skips to a link that says "see more" it will provide no context as to what that link represents. We should add the context even if it's visually-hidden, though better would be to restore the original link text that made it clear for all users.
